scripts/apps/authoring/versioning/history/history.js

```diff
@@ -30,7 +30,9 @@ function HistoryController($scope, authoring, api, notify, desks, archiveService
                             $scope.openVersion($scope.last);
                         }
                     }
-                });
+                }, function(err) {
```

Comment:
When we have an 'ingested' item being previewed, it has no versions and the error occurs. If we do not set $scope.versions to null in this case, the previous item's history is not removed from the display.
Also, currently the change affects items with all states ( not only the published ones ). Kindly let me know if this is OK or we need to have this work done strictly for published items only.